setup.py

```diff
@@ -34,15 +34,13 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
         "Framework :: Django",
-        "Framework :: Django :: 3.0",
         "Framework :: Django :: 3.2",
         "Framework :: Django :: 4.0",
         "Framework :: Django :: 4.1",
         "Framework :: Wagtail",
-        "Framework :: Wagtail :: 2",
-        "Framework :: Wagtail :: 3.0",
-        "Framework :: Wagtail :: 4.0",
+        "Framework :: Wagtail :: 4",
```

Comment:
This wasn't right either. I tend to follow the lead from [wagtailmedia package](https://github.com/torchbox/wagtailmedia/blob/cafdf367b0de76c2d6a1c6bf91d2fc57a153f77b/setup.py#L54)